test/automation/src/positron/positronDataExplorer.ts

```diff
@@ -58,31 +58,27 @@ export class PositronDataExplorer {
 	 */
 	async getDataExplorerTableData(): Promise<object[]> {
 
-		// unreliable:
-		//await this.code.waitForElement(IDLE_STATUS);
-		await this.code.driver.getLocator(IDLE_STATUS).waitFor({ state: 'visible', timeout: 60000 });
+		await this.code.driver.page.locator(IDLE_STATUS).waitFor({ state: 'visible', timeout: 60000 });
```

Comment:
you don’t have to change this since it’s a personal preference (and honestly, either way works the same i believe), but i lean towards this style because it feels clearer and more intuitive for someone quickly scanning through test steps:
`await expect(this.code.driver.page.locator(IDLE_STATUS)).toBeVisible({ timeout: 60000 })`

Reply:
Works for me :)